english/find-exit/into-forest/talk-to-cougar/talk-to-cougar.md

```diff
@@ -14,8 +14,6 @@ You ask Sara how she can speak to humans
 
 She says "Be careful asking questions you don't already know the answer to"
 
-You decide you in fact do not want to know the answer to the question,
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.